modules/claims_api/config/schemas/v2/2122a.json

```diff
@@ -247,8 +247,7 @@
       "additionalProperties": false,
       "required": [
         "poaCode",
-        "firstName",
-        "lastName",
```

Comment:
we can get these properties from the database with the registration number lookup